templates/index.html

```diff
@@ -1,8 +1,7 @@
 {% extends "base.html" %}
 <!--External links are differentiated on home page.-->
 {% block templatestyles %}
-<!-- a[href*="//"]:not([href*="{{config.base_url | safe}}"])::after {
-content: " ↗"; -->
+
 }
```

Comment:
This `}` needs to be removed, will show on the site. Remove the comment on line 2 as well just to be tidy.

Reply:
ah got it, good catch